packages/react/src/Accordion/AccordionContext.tsx

```diff
@@ -4,11 +4,12 @@
  */
 
 import { createContext } from 'react'
+import type { AccordionProps } from './Accordion'
 import type { HeadingProps } from '../Heading/Heading'
 
 export type AccordionContextValue = {
   headingLevel: HeadingProps['level']
-  sectionAs?: 'div' | 'section'
+  sectionAs?: AccordionProps['sectionAs']
 }
 
 const defaultValues: AccordionContextValue = {
```

Comment:
There’s still a value of 1 for `headingLevel` in this object. (How) can we remove it?

Reply:
I’ve added `null` to the type in Context.